src/app/pages/ledger/connect-ledger/connect-ledger.layout.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Stack } from '@stacks/ui';
+
+import { Divider } from '@app/components/divider';
+import { BaseDrawer } from '@app/components/drawer';
+import { Link } from '@app/components/link';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Caption } from '@app/components/typography';
+import ConnectLedger from '@assets/images/onboarding/ledger/connect-ledger.png';
+// import { WarningLabel } from '@app/components/warning-label';
+
+import { LookingForLedger } from './components/looking-for-ledger';
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface ConnectLedgerLayoutProps {
+  isLookingForLedger: boolean;
+  onCancelConnectLedger(): void;
+  onConnectLedger(): void;
+}
+export function ConnectLedgerLayout(props: ConnectLedgerLayoutProps) {
+  const { isLookingForLedger, onCancelConnectLedger, onConnectLedger } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
+      <Stack alignItems="center" pb="loose" px="loose" spacing="loose" textAlign="center">
+        <Stack spacing="extra-loose" width="331px">
+          {/* TODO: Implement warning with actual ledger integration */}
+          {/* <WarningLabel>Before proceeding, close Ledger Live if it’s open</WarningLabel> */}
+          <Box mt="tight">
+            <img src={ConnectLedger} />
+          </Box>
+        </Stack>
+        <ConnectLedgerTitle />
+        {isLookingForLedger ? (
+          <LookingForLedger />
+        ) : (
+          <>
+            <PrimaryButton height="40px" onClick={onConnectLedger}>
+              Connect
+            </PrimaryButton>
+            <Divider />
+            <Box>
+              <Caption mb="extra-tight">First time using Ledger on Hiro Wallet?</Caption>
```

Comment:
This renders a `span` which is by default an inline element, and on the css we are defining it as a `block` level element. 

Why don't we just render a `div` instead?

Reply:
Noticed this too. Any reason why we need any wrapper around `<Caption />`?

Reply:
I'd indeed avoid using `<Stack />` here. Checking the design, the elements don't all have exactly the same spacing. So stack kinda sets a default margin, then you need to adjust margins in a handful of places anyway.

Would recommend using a Flex, and manually set `mt` properties in every case.